opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -596,6 +596,10 @@ opennms.eventlist.showCount=false
 #opennms.poller.server.serverPort=1199
 #opennms.poller.server.registryPort=1099
 
+# The enablePollStatusEvent is a flag that globally enables sending a special
```

Comment:
Can we also add a note that warns users about the potential impact when enabling this on a large system monitoring thousands of services?